hail_scripts/v02/utils/hail_utils.py

```diff
@@ -0,0 +1,123 @@
+import hail as hl
```

Comment:
I think this module could use a better name. `hail_utils` nested in `hail_scripts.utils` is a bit redundant. Also maybe splitting up... `run_vep` is definitely unlike the other functions here, which are all import/export related.

Reply:
sure .. happy to go with your preferences on the dir structure.